utilities/rpp-unittests/HIP_NEW/Tensor_hip_pkd3.cpp

```diff
@@ -2364,7 +2364,7 @@ int main(int argc, char **argv)
         count += dstDescPtr->strides.nStride;
 
         char temp[1000];
-        strcpy(temp, dst);
+        strcpy(temp, "dst");
```

Comment:
need to use std::strings instead of old c style coding

Reply:
I will clean this up Rajy.